pkgs/tools/misc/fclones/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, rustPlatform, fetchgit, pkg-config, udev }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fclones";
+  version = "0.14.0";
+
+  src = fetchgit {
+    url = "https://github.com/pkolaczk/${pname}";
+    rev = "v${version}";
+    sha256 = "1ni5amy903cy822yhw070bcrrixrw2m1vr66q1h32bc98pyv4w05";
+  };
+
+  cargoSha256 = "1gcb46k7bwdfsf6hyvmi6dna1nf6myzy63bhjfp0wy7c8g4m2mg8";
+
+  nativeBuildInputs = [ pkg-config udev ];
```

Comment:
`udev` should probably be in `buildInputs`?

Reply:
Changed. I need to better understand the difference...

Reply:
`buildInputs` is usually for stuff like libraries and `nativeBuildInputs` is usually for stuff like build tools